cypress/objects/pageComponents.js

```diff
@@ -0,0 +1,22 @@
+class PageComponents {
+    getUrl(){
+        return cy.url()
+    }
+
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+
+    getLoggedInUser(){
+        return cy.get('.user-info__title')
+    }
+
+    getCalendar(){
```

Comment:
I don't calendar should be in page components, it should be in timeLogging page objects